be/src/util/runtime_profile.h

```diff
@@ -679,6 +679,39 @@ class ScopedRawTimer {
     int64_t* _counter;
 };
 
+// Utility class to update time elapsed manually
+// 'T' must implement the stopWatch "interface" (start,stop,elapsed_time) but
+// we use templates not to pay for virtual function overhead.
+template <class T>
+class StageTimer {
```

Comment:
utility is not added unless it is required, use Counter and StopWatch just can do same work. and the code is even more clear, since Counter and StopWatch are used in present code widely.

Reply:
removed, fixed